docs/orleans/overview.md

```diff
@@ -20,19 +20,13 @@ It was created by [Microsoft Research](https://research.microsoft.com/projects/o
 
 ## Grains
 
-<!--
-![A grain is composed of a stable identity, behavior, and state](~/images/grain_formulation.svg)
--->
+:::image type="content" source="media/grain-formulation.svg" alt-text="A grain is composed of a stable identity, behavior, and state.":::
 
 The fundamental building block in any Orleans application is a *grain*. Grains are entities comprising user-defined identity, behavior, and state. Grain identities are user-defined keys that make grains always available for invocation. Grains can be invoked by other grains or by external clients such as Web frontend via strongly-typed communication interfaces (contracts). Each grain is an instance of a class that implements one or more of these interfaces.
 
 Grains can have volatile or persistent state data that can be stored in any storage system. As such, grains implicitly partition application states, enabling automatic scalability and simplifying recovery from failures. Grain state is kept in memory while the grain is active, leading to lower latency and less load on data stores.
 
-<!--
-    <p align="center">
-      ![](~/images/managed_lifecycle.svg)
-    </p>
--->
+:::image type="content" source="media/managed-lifecycle.png" alt-text="The managed lifecycle of an Orleans grain.":::
```

Comment:
You can use SVG if you want: https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-create-conceptual-art?branch=main#file-types

Reply:
I know, but their SVG was not salvageable. So, I re-drew it in Visio.